sdk/web-pubsub/web-pubsub/test/testEnv.ts

```diff
@@ -8,6 +8,8 @@ const envSetupForPlayback: Record<string, string> = {
   WPS_API_KEY: "api_key",
   WPS_ENDPOINT: "https://endpoint",
   WPS_REVERSE_PROXY_ENDPOINT: "https://endpoint",
+  WPS_SOCKETIO_CONNECTION_STRING: "Endpoint=endpoint;AccessKey=api_key;Version=1.0;",
```

Comment:
I think there was an effort removing connection string dependency from the tests

Reply:
you're right, let me remove it